release.yaml

```diff
@@ -78,6 +78,10 @@ rancher-cis-benchmark-crd:
 system-upgrade-controller:
 - 100.0.2+up0.3.2
 rancher-cis-benchmark:
-- 2.0.3
+- 2.0.3-rc1
 rancher-cis-benchmark-crd:
-- 2.0.3
+- 2.0.3-rc1
+rancher-gatekeeper:
+- 100.0.1+up3.6.0
+rancher-gatekeeper-crd:
+- 100.0.1+up3.6.0
```

Comment:
these versions have already been released, by adding them to the release.yaml you're basically saying that you want to modify an already released chart (which is not what you are intending; you want to create a new release).

package.yaml needs to be bumped to 100.0.2